tests/test_interface.rb

```diff
@@ -958,6 +959,42 @@ def test_interface_ipv4_arp_timeout
     assert_raises(RuntimeError) { nonvlanint.ipv4_arp_timeout = 300 }
   end
 
+  def test_interface_fabric_forwarding_anycast_gateway
+    # Setup
+    config('no interface vlan11')
+    int = Interface.new('vlan11')
+    foo = OverlayGlobal.new
+    foo.anycast_gateway_mac = '1223.3445.5668'
+
+    # 1. Testing default
+    int.fabric_forwarding_anycast_gateway =
+      int.default_fabric_forwarding_anycast_gateway
```

Comment:
I'd prefer removing these two lines - have the test below be for verifying that the default state of a newly created Vlan matches the expected default value, rather than explicitly setting it to default.